cypress/e2e/user.cy.js

```diff
@@ -2,11 +2,29 @@
 /// <reference types='../support' />
 
 describe('User', () => {
+  let user;
+
   before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.register();
+    cy.login();
   });
 
-  it.skip('should be able to follow the another user', () => {
+  it('should be able to follow/unfollow the another user', () => {
+    cy.visit('#/@riot');
+    cy.get('class="btn btn-sm btn-outline-secondary action-btn"').click();
```

Comment:
The selector `class="btn btn-sm btn-outline-secondary action-btn"` is incorrect. It should be `.btn.btn-sm.btn-outline-secondary.action-btn` to correctly select the element by class names.